index.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+require_once "Autor.php";
+require_once "ImprimirAutor.php";
+require_once "Revista.php";
+require_once "Libro.php";
+
+$autorElenaWhite = new Autor('Elena G. de White', 'Americana');
+echo $autorElenaWhite->getInfo();
+
+$autorCesarCondori = new Autor("Cesar Daniel Condori Malpartida", "Brasilero");
+echo $autorCesarCondori->getInfo();
+
+$autorIsaacSurci = new Autor('Isaac Surci', 'Boliviano');
+echo $autorIsaacSurci->getInfo();
+
+$autorLuisChura = new Autor('Luis Chura', 'Boliviano');
+echo $autorLuisChura->getInfo();
+
+$ImprimirAutor = new ImprimirAutor();
```

Comment:
```suggestion
$imprimirAutor = new ImprimirAutor();
```
La primera letra en minúscula para variables

Reply:
cierto